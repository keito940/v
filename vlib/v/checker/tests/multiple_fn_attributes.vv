[inline;deprecated]
fn foo(name string) string {}