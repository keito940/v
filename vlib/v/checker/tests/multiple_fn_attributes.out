vlib/v/checker/tests/multiple_fn_attributes.v:2:1: error: multiple attributes detected 
    1 | [inline;deprecated]
    2 | fn foo(name string) string {}
      | ~~